parm/config/config.anal

```diff
@@ -57,21 +57,22 @@ export OBERROR=$FIXgsi/prepobs_errtable.global
 
 
 # Use experimental dumps in EMC GFS v16 parallels
+export COMPONENT=${COMPONENT:-atmos}
 if [[ $RUN_ENVIR == "emc" ]]; then
     export ABIBF="/dev/null"
     if [[ "$CDATE" -ge "2019022800" ]] ; then
-	export ABIBF="$DMPDIR/${CDUMP}x.${PDY}/${cyc}/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
+	export ABIBF="$DMPDIR/${CDUMP}x.${PDY}/${cyc}/${COMPONENT}/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
```

Comment:
I don't see why these can't be wired to `atmos` since this is the atmospheric analysis and obs.

Reply:
I accepted the change to hardcode `COMPONENT=atmos` before this line in `config.anal` so this will be `atmos` now via the variable. Want me to hardcode to `atmos` here still?